addons/l10n_es_pos/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Spain - Point of Sale',
+    'countries': ['es'],
+    'author': 'Odoo, Odoo Community Association (OCA)',
+    'category': 'Accounting/Localizations/Point of Sale',
+    'summary': """Spanish localization for Point of Sale""",
+    'depends': ['point_of_sale', 'l10n_es'],
+    'auto_install': True,
+    'license': 'LGPL-3',
+    'post_init_hook': '_post_init_hook',
```

Comment:
could you rename it and do something like:
"_l10n_es_pos_xxxxxxxxxxxx_post_init_hook"